build/travis-debug/Dockerfile

```diff
@@ -0,0 +1,53 @@
+# Simulate our Maven build on Travis-CI
+# WORKDIR ends up as /home/travis/build
+# Source directory should be mounted here
+
+FROM quay.io/travisci/travis-jvm
```

Comment:
That's the [Travis-provided image that they use in production](https://docs.travis-ci.com/user/common-build-problems/#Running-a-Container-Based-Docker-Image-Locally).